client/coral-embed-stream/src/Embed.js

```diff
@@ -124,6 +124,16 @@ class Embed extends Component {
             <Tab>{lang.t('MY_COMMENTS')}</Tab>
             <Tab restricted={!isAdmin}>Configure Stream</Tab>
           </TabBar>
+          {
+            highlightedComment &&
+            <Button
```

Comment:
This should be an isolated component. We can refator this when we refactor the Embed.